<launch>
	<arg name="rosbag_dir" default="$(env HOME)/rosbag/airsim/drone_2cam_lidar16"/>
	<arg name="rosbag_name" default="2022-10-09-12-58-54"/>

	<!-- rviz -->
	<node pkg="rviz" type="rviz" name="rviz" args="-d $(find rosbag_to_dataset)/rviz_config/debug.rviz"/>

	<!-- main -->
	<node pkg="rosbag_to_dataset" type="rosbag_to_dataset" name="rosbag_to_dataset" output="screen">
		<param name="rosbag_path" type="string" value="$(arg rosbag_dir)/$(arg rosbag_name).bag"/>
		<param name="save_dir" type="string" value="$(find rosbag_to_dataset)/save/$(arg rosbag_name)"/>
		<param name="debug_hz" type="double" value="10.0"/>
		<param name="min_time_diff_sec" type="double" value="1.0"/>
		<!-- <param name="min_odom_diff_m" type="double" value="1.0"/> -->
		<!-- <param name="min_odom_diff_deg" type="double" value="30.0"/> -->
		<param name="topic_0" type="string" value="/airsim_node/drone/Camera0/Scene/compressed"/>
		<param name="topic_1" type="string" value="/airsim_node/drone/Camera90/Scene/compressed"/>
		<param name="topic_2" type="string" value="/airsim_node/drone/lidar/LidarCustom"/>
		<param name="topic_3" type="string" value="/airsim_node/drone/imu/Imu"/>
		<param name="topic_4" type="string" value="/airsim_node/drone/odom_local_ned"/>
	</node>
</launch>